test/k8sT/Nightly.go

```diff
@@ -388,6 +388,39 @@ var _ = Describe("NightlyExamples", func() {
 		})
 
 	})
+	Context("Upgrade test", func() {
```

Comment:
In the future, please do the `Context` addition / whitespace changes as a separate commit, which contains no code changes. This relegates all whitespace changes to one commit, and allows for easier review of the actual, functional changes in the PR.